### ISSUE TITLE
SlevomatCodingStandard.Namespaces.UnusedUses false positive in HEREDOC

### ISSUE THREAD

**Issue by yguedidi** [slevomat/coding-standard#681]
Here a broken case:

```php
use \Namespace\Foo;

$foo = <<<TXT
some text {Foo::class}
TXT;
```

`Foo` is detected as unused

**Comment by kukulich** [comment]
I's very complicated to solve it and because it's very rare case I will not fix it.
